src/main/twirl/gitbucket/core/issues/issueinfo.scala.html

```diff
@@ -210,7 +210,8 @@
       $('#label-milestone').html($('<span class="muted small">').text('No milestone'));
       $('#milestone-progress-area').empty();
     } else {
-      $('#label-milestone').html($('<span class="strong small">').text(title));
+      $('#label-milestone').html($('<a class="strong small username">').text(title)
+        .attr('href', '@helpers.url(repository)/issues?milestone=' + title + '&state=open'));
```

Comment:
Same as above, however you can't use `helpers.urlEncode()` because here is JavaScript world.